src/accelerate/accelerator.py

```diff
@@ -2027,6 +2030,13 @@ def prepare_optimizer(self, optimizer: torch.optim.Optimizer, device_placement=N
         >>> optimizer = accelerator.prepare_optimizer(optimizer, device_placement=True)
         ```
         """
+        if is_lomo_available():
+            # We need to import locally to avoid circular imports since lomo imports stuff from
+            # transformers & accelerate
+            from lomo_optim import AdaLomo, Lomo
+
+            self.has_lomo_optimizer = isinstance(optimizer, (Lomo, AdaLomo))
```

Comment:
For my understanding, we assume that the accelerator could have multiple `self._optimizers`, with some of them LOMO and others not. Would that not create the issue that `self.has_lomo_optimizer` takes the value based on whatever the last optimizer is? Would we not have to set: `self.has_lomo_optimizer |= isinstance(optimizer, (Lomo, AdaLomo))`